src/App.tsx

```diff
@@ -66,6 +67,7 @@ function App() {
           code: currentFileContent
         })
       });
+
       const responseData = await response.json();
       setCompilingStatus(false);
       if (responseData.error) {
```

Comment:
Shouldn't we add await before setCompilationError(responseData.error); at line 74:

Reply:
Same as for runContractCompilation. We don't have any logic that needs to await for it and also it's react useState hook I am not sure if it's possible to await.